pkg/network/tracer/tracer_test.go

```diff
@@ -1253,3 +1254,60 @@ func (s *TracerSuite) TestTCPDirection() {
 	conn = incomingConns[0]
 	assert.Equal(t, conn.Direction, network.INCOMING, "connection direction must be incoming: %s", conn)
 }
+
+func (s *TracerSuite) TestTCPFailureConnectionRefused() {
+	t := s.T()
+
+	checkSkipFailureConnectionsTests(t)
```

Comment:
this will skip the Refused test on older kernels on linux, which we only want to do for the timeout test

Reply:
🙇